run-snapshot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This script builds the documentation in a local checkout of the kroxylicious/kroxylicious repository, then incorporates
+# the built documentation into a local deployment of the website and serves it on localhost:4000 using ./run.sh.
+#
+# It is assumed this repository is checked out beside a cloned https://github.com/kroxylicious/kroxylicious/ like:
+# .
+# ├── kroxylicious
+# └── kroxylicious.github.io
+# 
+# if you organize your directories differently, you can set the KROXYLICIOUS_DIR env variable like
+# KROXYLICIOUS_DIR=/path/to/my/kroxylicious ./run-snapshot.sh to point it at your kroxylicious dir.
+
+trap "exit" INT
+set +euo pipefail
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+KROXYLICIOUS_DIR="${KROXYLICIOUS_DIR:-${SCRIPT_DIR}/../kroxylicious}"
```

Comment:
You could test whether this exists (and contains a pom.xml) and exit with a nice error message if not.